app/(dashboard)/profile/page.tsx

```diff
@@ -49,7 +49,11 @@ const ProfilePage = async () => {
                 className='rounded-full'
               />
             )}
-            <ProfileEdit userId={user.id} user={userProfile.data.username} />
+            <ProfileEdit
+              userId={user.id}
+              user={userProfile.data.username}
```

Comment:
`userName` to keep in line with the pattern used for props, but also to avoid confusion as `user` seems like it would take a user object